src/Sulu/Bundle/WebsiteBundle/Admin/WebsiteAdmin.php

```diff
@@ -44,4 +49,21 @@ public function getJsBundleName()
     {
         return 'suluwebsite';
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSecurityContexts()
+    {
+        return [
+            'Sulu' => [
+                'Website' => [
+                    'sulu.website.cache' => [
```

Comment:
Since (correct me if I am wrong) the cache can currently be only purged for every webspace, I would suggest to only show these UI items if the user has the `PUBLISH` (or maybe `EDIT` since `PUBLISH` doesn't really exist) permission for all the webspaces.